py_mmd_tools/mmd_elements.yaml

```diff
@@ -154,12 +198,14 @@ geographic_extent:
           maxOccurs: '1'
           minOccurs: '1'
           comment: No repetition allowed.
+          description: Describes a simple longitude limit; may be part of a 2- or 3-dimensional bounding region. geospatial_lon_min specifies the westernmost longitude covered by the dataset. See also geospatial_lon_max. Must be decimal degrees east (negative westwards).
   polygon:
     maxOccurs: '1'
     minOccurs: '0'
     acdd:
       geospatial_bounds:
-        comment: No repetition allowed.
+        comment: Recommended.
```

Comment:
Earlier ( See Line 66) you use recommended: False.
This should probably be recommended: True
?

Reply:
yes - I'll set this as an empty string